resfulservice/package.json

```diff
@@ -38,6 +38,7 @@
         "swagger-ui-express": "^4.6.3",
         "ua-parser-js": "^1.0.33",
         "unique-names-generator": "^4.7.1",
+        "uuid": "^9.0.1",
```

Comment:
This isn't a request for a change here, just a personal source of confusion. Am curious why we include `uuid` in the resfulservice/package.json but not in in the app/package.json (since we also use it in the SDD form logic). **_Should_** we be including `uuid` in the app/package.json?

Reply:
Sure, the restful now uses it in its interactions with the managed services container.